cli/gluster-block.c

```diff
@@ -486,8 +486,8 @@ glusterBlockModify(int argcount, char **options, int json)
     if(ret >= 0) {
       mobj.auth_mode = ret;
     } else {
-      MSG(stderr, "%s\n", "'auth' option is incorrect");
-      MSG(stderr, "%s\n", GB_MODIFY_HELP_STR);
+      MSG(stderr, "'auth' option is incorrect");
+      MSG(stderr, "%s", GB_MODIFY_HELP_STR);
```

Comment:
I mean here:

No need of '%s' in MSG() according to your change, I see you just removed '\n' in many more places, please remove %s in all of them.

We can change this to:
MSG(stderr, GB_MODIFY_HELP_STR);

Reply:
Yeah, make sense.
Will fix it.
Thanks.